src/storage/src/hummock/file_cache/metrics.rs

```diff
@@ -15,79 +15,109 @@
 use std::sync::Arc;
 
 use prometheus::{
-    register_counter_with_registry, register_histogram_with_registry,
+    register_counter_vec_with_registry, register_histogram_vec_with_registry,
     register_int_counter_with_registry, Counter, Histogram, IntCounter, Registry,
 };
 
 pub struct FileCacheMetrics {
+    pub cache_miss: IntCounter,
+
     pub disk_read_throughput: Counter,
     pub disk_read_latency: Histogram,
     pub disk_write_throughput: Counter,
```

Comment:
minor: should we call it `disk_write_total_bytes` instead? It is actually a counter for total bytes written and the throughput is derived from it.